rclcpp/topics/minimal_publisher/member_function_with_wait_for_all_acked.cpp

```diff
@@ -30,7 +30,7 @@ class MinimalPublisher : public rclcpp::Node
   MinimalPublisher()
   : Node("minimal_publisher_with_wait_for_all_acked"),
     count_(0),
-    wait_timeout_(300)
+    wait_timeout_(-1)
```

Comment:
see my comment https://github.com/ros2/rmw_fastrtps/issues/822#issuecomment-2867977437

@Barry-Xu-2018 @MiguelCompany do you think any downside of this?

Reply:
@fujitatomoya 

Maybe we can set a longer wait time. Otherwise, the code from line 61-66 needs to be removed because when the timeout is -1, the else part of the code will never be executed.

Reply:
yeah i though that too. longer timeout could be safer than infinite timeout? if you prefer that, how about 5 seconds? i am almost sure, with 5 seconds this problem should never happen unless something else goes wrong.